Gemfile

```diff
@@ -17,13 +17,16 @@ group :development do
 end
 
 group :test do
-  gem 'rspec'
   gem 'rack-test'
   gem 'webmock'
   gem 'simplecov', :require => false
   gem 'rest-client'
 end
 
+group :test, :development do
```

Comment:
The rspec gem is not needed to be loaded in development environment. Add a condition in the Rakefile to check when the RACK_ENV is test